src/sentry/static/sentry/app/components/stackedBarChart.tsx

```diff
@@ -114,36 +119,46 @@ class StackedBarChart extends React.Component<Props, State> {
 
     // massage points
     let series = props.series;
+    let secondarySeries: Series = [];
 
     if (props.points?.length) {
       if (series?.length) {
         throw new Error('Only one of [points|series] should be specified.');
       }
 
       series = this.pointsToSeries(props.points);
+      if (props.secondaryPoints.length)
```

Comment:
nit: even if it's just a one line if statement can we still use brackets '{'?

Reply:
I'm impartial so sure